service.iptv.manager/resources/lib/kodilogging.py

```diff
@@ -8,37 +8,51 @@
 import xbmc
 import xbmcaddon
 
+from resources.lib import kodiutils
+
+ADDON = xbmcaddon.Addon()
+
 
 class KodiLogHandler(logging.StreamHandler):
-    """A log handler for Kodi"""
+    """ A log handler for Kodi """
 
     def __init__(self):
         logging.StreamHandler.__init__(self)
-        addon_id = xbmcaddon.Addon().getAddonInfo("id")
-        formatter = logging.Formatter("[{}] [%(name)s] %(message)s".format(addon_id))
+        formatter = logging.Formatter("[{}] [%(name)s] %(message)s".format(ADDON.getAddonInfo("id")))
         self.setFormatter(formatter)
+        # xbmc.LOGNOTICE is deprecated in Kodi 19 Matrix
+        if kodiutils.kodi_version_major() > 18:
```

Comment:
The min. debug level ought to be xbmc.LOGDEBUG to avoid spamming logs.

Reply:
We only set this in case the "debug logging" setting is activated in the settings, to be able to troubleshoot issues. By default, we aren't logging debug messages in the Kodi log. See line 43 of this file.

In Matrix, the minimum level that causes something to end up in the log has changed from NOTICE to INFO, that's why this is here.